dependencies.props

```diff
@@ -18,6 +18,7 @@
 
   <PropertyGroup>
     <CoreFxVersion>4.5.0-preview1-25712-02</CoreFxVersion>
+    <CoreFxUapVersion>4.6.0-preview1-25712-02</CoreFxUapVersion>
```

Comment:
I think you need to change https://github.com/dotnet/core-setup/blob/95ed2ce31b4a7f33fbea1afcd5962c7fef789823/src/pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/src/Microsoft.NETCore.UniversalWindowsPlatform.depproj#L26-L28 to use this.

Reply:
doh